docs/rendering/modelextensions/rendertypes.md

```diff
@@ -0,0 +1,78 @@
+Render Types
+============
+
+Adding the `render_type` entry at the top level allows specifying the render type this model will render in.  
+If this is not specified, the model will fall back to the render types returned by `ItemBlockRenderTypes#getRenderLayers()`.  
+This is merely a hint to the model loader and custom model loaders are free to ignore this entry.
+
+!!! note
+    Since 1.19 this is preferred over the deprecated method of setting the applicable render type(s) via `ItemBlockRenderTypes#setRenderLayer()` for blocks.
+
+Example of a model for a cutout block with the glass texture
+
+```js
+{
+  "render_type": "minecraft:cutout",
+  "parent": "block/cube_all",
+  "textures": {
+    "all": "block/glass"
+  }
+}
+```
+
+Vanilla Values
+--------------
+
+The following options with the respective chunk and entity render type are supplied by Forge (`NamedRenderTypeManager#preRegisterVanillaRenderTypes()`):
+
+* `minecraft:solid`
+    * Chunk render type: `RenderType#solid()`
+    * Entity render type: `ForgeRenderTypes#ITEM_LAYERED_SOLID`
+    * Used for fully solid blocks (i.e. Stone)
+* `minecraft:cutout`
+    * Chunk render type: `RenderType#cutout()`
+    * Entity render type: `ForgeRenderTypes#ITEM_LAYERED_CUTOUT`
+    * Used for blocks where any given pixel is either fully transparent or fully opaque (i.e. Glass Block)
+* `minecraft:cutout_mipped`
+    * Chunk render type: `RenderType#cutoutMipped()`
+    * Entity render type: `ForgeRenderTypes#ITEM_LAYERED_CUTOUT`
+    * Chunk and entity render type differ due to mipmapping on the entity render type making items look weird
+    * Used for blocks where any given pixel is either fully transparent or fully opaque and the texture should be scaled down at larger distances ([mipmapping]) to avoid visual artifacts (i.e. Leaves)
+* `minecraft:cutout_mipped_all`
+    * Chunk render type: `RenderType#cutoutMipped()`
+    * Entity render type: `ForgeRenderTypes#ITEM_LAYERED_CUTOUT_MIPPED`
+    * Used in similar cases as `minecraft:cutout_mipped` when the item representation should also have mipmapping applied
+* `minecraft:translucent`
+    * Chunk render type: `RenderType#translucent()`
+    * Entity render type: `ForgeRenderTypes#ITEM_LAYERED_TRANSLUCENT`
+    * Used for blocks where any given pixel may be partially transparent (i.e. Stained Glass)
+* `minecraft:tripwire`
+    * Chunk render type: `RenderType#tripwire()`
+    * Entity render type: `ForgeRenderTypes#ITEM_LAYERED_TRANSLUCENT`
+    * Chunk and entity render type differ due to the tripwire render type not being feasible as an entity render type
```

Comment:
In all cases except the two I mention it specifically (the referenced one being one of those), the entity render type is the direct entity-equivalent of the chunk render type, but in these two cases it's a different one for a specific reason in case of the cutout_mipped render type and for a reason I have to assume in case of the tripwire